catroid/src/main/java/org/catrobat/catroid/content/bricks/ShowTextBrick.java

```diff
@@ -187,10 +187,25 @@ public void onNothingSelected(AdapterView<?> arg0) {
 	@Override
 	public View getPrototypeView(Context context) {
 		prototypeView = View.inflate(context, R.layout.brick_show_variable, null);
-		TextView textViewX = (TextView) prototypeView.findViewById(R.id.brick_show_variable_edit_text_x);
-		textViewX.setText(Utils.getNumberStringForBricks(BrickValues.X_POSITION));
-		TextView textViewY = (TextView) prototypeView.findViewById(R.id.brick_show_variable_edit_text_y);
-		textViewY.setText(Utils.getNumberStringForBricks(BrickValues.Y_POSITION));
+
+		Spinner variableSpinner = (Spinner) prototypeView.findViewById(R.id.show_variable_spinner);
```

Comment:
duplicate code, can this be moved somewhere general for all user variable bricks?